Userland/Utilities/headless-browser.cpp

```diff
@@ -352,6 +352,12 @@ static ErrorOr<TestResult> run_ref_test(HeadlessWebContentView& view, StringView
 
 static ErrorOr<TestResult> run_test(HeadlessWebContentView& view, StringView input_path, StringView expectation_path, TestMode mode, bool dump_failed_ref_tests)
 {
+    // Clear the current document.
+    // FIXME: Implement a debug-request to do this more thoroughly.
+    view.on_load_finish = {};
+    view.on_text_test_finish = {};
+    view.load(URL("about:blank"sv));
```

Comment:
could you make it wait for "about:blank" to load?

Reply:
If we just want to wait, you can block on a promise:
```c++
auto promise = Core::Promise<Empty>::construct();

view.on_load_finish = [&]() {
	promise->resolve({});
};

view.load(URL("about:blank"sv));
MUST(promise->await());
```

Reply:
Great! Thank you Tim!